LIBITD-2566. Stying Updates for IPManager

am486dx opened the pull request:
Added shared styles used by IPManager and likely to be reused in other applications. This includes button-style links, login/ publish buttons, default margins, and SAML login notice.

https://umd-dit.atlassian.net/browse/LIBITD-2566

am486dx committed:
LIBITD-2566. Stying Updates for IPManager

Added shared styles used by IPManager and likely to be reused
in other applications. This includes button-style links, login/
publish buttons, default margins, and SAML login notice.

https://umd-dit.atlassian.net/browse/LIBITD-2566

peichman-umd committed:
LIBITD-2566. Updates to button and button link styling.

- changed the selector for button-styled links from `.link-button` and `.link-button--{action}` to `a.button` and `a.button.{action}`
- grouped `button.{action}` and `a.button.{action}` selectors
- grouped `:hover` selectors with their base selectors
- added some more variables for repeated values (border-radius, widget fonts, sizes, and paddings, etc.)

https://umd-dit.atlassian.net/browse/LIBITD-2566

am486dx committed:
Merge pull request #1 from peichman-umd/feature/LIBITD-2566

LIBITD-2566. Updates to button and button link styling.

am486dx commented:
Anything else you'd want modified? @peichman-umd

peichman-umd commented:
@am486dx I can add the alternate table styles that I developed for the archelon-django R&D project that are a little lighter visually than the Grove ones (and closer in appearance to the Bootstrap default table styles).

peichman-umd committed:
LIBITD-2566. Expanded table styles.

- moved the current default table styling (originated in Grove) to the `table.grid` class
- added a `table.lined` lighter weight table display style that is similar to Bootstrap's default table styling
- added more table-related variables

https://umd-dit.atlassian.net/browse/LIBITD-2566

peichman-umd committed:
LIBITD-2566. Added selector to use to single button forms display inline.

https://umd-dit.atlassian.net/browse/LIBITD-2566

am486dx commented:
@peichman-umd I'm not too invested in it either way - if you think adding them to the project would be preferable I'm happy to do that as well.

peichman-umd commented:
@am486dx I've created this PR to add more table styling options, plus a selector for an inline single-button form: https://github.com/am486dx/django-umd-lib-style/pull/2

am486dx committed:
Merge pull request #2 from peichman-umd/feature/LIBITD-2566

LIBITD-2566. Table and single-button form styles.

am486dx commented:
@peichman-umd merged in!
